### PR TITLE
Add present on 0.5.0 skygear release

### PR DESCRIPTION
**Pull request by rickmak**

### CODE REVIEW

File: skygear-0.5.0.slide [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Rick Mak
+Oursky
+rickmak@oursky.com
+@rickmak
+CC81327D
+
+* Important Changes
+
+- zmq is now depreacted
+- `http` and `exec` will remove c deps
```
Comment (cheungpat) [inline skygear-0.5.0.slide]: s/will remove/does not have/

File: skygear-0.5.0.slide [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+You can now directly install by
+
+    go install github.com/oursky/skygear
```
Comment (cheungpat) [inline skygear-0.5.0.slide:18]: Not sure if you want to mention that skygear can be downloaded from https://github.com/oursky/skygear/releases

File: skygear-0.5.0.slide [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+Say goodbye to Docker, in development
+
+* Incoporate common usecase in Core
```
Comment (cheungpat) [inline skygear-0.5.0.slide]: s/Incoporate/Incorporate/
s/usecase/use case/
s/Core/Server/

**Comment by rickmak** [comment]
Merged to master via 52b0567  and ce160ed

**Pull request closed without merging**